qds_sdk/commands.py

```diff
@@ -1345,3 +1371,70 @@ def _is_complete_data_available(bucket_paths, num_result_dir):
                 one_path.get_contents_to_file(fp)  # cb=_callback
             else:
                 _read_iteratively(one_path, fp, delim=delim)
+
+
+def _download_to_local_oracle(storage_conn, oracle_path, fp, delim=None,skip_data_avail_check=False):
```

Comment:
Yes I will move this function to cloud.rb in v2 changes to make it cloud agnostic. If I change it here , then again I have to change in v2 api too.